### PR TITLE
Rework on the documentation during the website sprint 1

### PR DESCRIPTION
**Pull request by vienlt**
Changes:

- Restructure the documentation

- Rewrite the "Register data" document

- Add the "Contribute to the Protocol" document

- Add some initial introduction for the new "Control Access" document

All the contents are updated with the latest content on the master.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,45 @@
 
 # Bitmark Documentation
 
-## [Learning Bitmark](/pages/learning-bitmark/README.md)
+## [Register & Control data](/pages/register-and-control-data/README.md)
```
Comment (shannona) [inline README.md:4]: It's not clear from this title what this really is. The problem is that "register" could be a noun (meaning a record) or a verb (meaning to record something). I think you mean the verb, and if so, I'd suggest "Registering & Controlling Data".

Comment (shannona) [reply README.md:4]: (Or looking at the contents, maybe "Registering Data & Controlling Access".

File: pages/browse-our-faq/bitmark-protocol-faq.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+keywords: faq
+last_updated: 
+sidebar: mydoc_sidebar
+permalink: /browser-our-faq/bitmark-protocol-faq
+folder: browser-our-faq
+---
+
+# Bitmark Protocol FAQ
+
+An early engineering fault with the Web was that information ownership and control was not addressed as an “end-to-end” internet protocol. This has allowed for all sorts of externalities to occur that range from digital rights management to fake news to mass-surveillance.
```
Comment (shannona) [inline pages/browse-our-faq/bitmark-protocol-faq.md:12]: I feel like these need more explanation, particularly to demonstrate how the problems are a result of the lack of secure data provenance.

Comment (shannona) [reply pages/browse-our-faq/bitmark-protocol-faq.md:12]: I also never particularly love the use of "externalities" in explanations (though I've done so myself). I don't think most people understand what it means.

File: pages/browse-our-faq/bitmark-protocol-faq.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+To alleviate the externalities of the Internet, we must have a system to control information. One option is to go “centralized” with government regulation and corporate Oversight Boards that are tasked with making new rules and then moderating content. However, this is a global issue, so State-level regulation will often conflict across different countries. The other option is to go “decentralized” and build an end-to-end protocol that secures the origin and history, or “provenance” of data itself. The benefits are:
+
+* When data has provenance, information (processed data) can be authenticated without a trusted third party. 
+* End-to-end protocols are low-cost.
+* Having a global data provenance enhances local regulation (more below).
+
+## Why hasn’t this been done before?
+
+End-to-end protocols cannot rely on trusted third parties. In order to have such a protocol for information ownership/control we must have the ability to secure the provenance of data. Information is processed data. Before Bitcoin (specifically public blockchains), there was not a way to secure the provenance of anything digital without relying upon trusted third parties. 
+
```
Comment (shannona) [inline pages/browse-our-faq/bitmark-protocol-faq.md:27]: Would suggest something like:

"End-to-end protocols cannot rely on trusted third parties. Instead, they require the ability to secure the provenance of data. This has only become possible with the advent of Bitcoin, and its underlying blockchain technology. With blockchains, digital provenance can be secured without relying upon trusted third parties."

File: pages/browse-our-faq/bitmark-protocol-faq.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+When changes or ownership / control are needed, the underlying data can be transferred peer-to-peer, creating an unforgeable provenance. 
+
+## How can this protect digital rights? 
+
+Information is the key to human rights. And the key to controlling information is property rights. 
+
+Property rights are agreements of control across different trust boundaries. For example, you could have a property right to access a river running through your land. In the digital environment, a tech company could recognize a property right to use your data in their algorithm. 
+
+For most of human history, the dominant form of property has been personal property. The functionality of personal property has not usually depended on trusted third parties. Today, an increasing share of our personal property is digital, and our rights depend on BigTech. An end-to-end protocol for information control could record digital property rights and provide a better future. 
+
```
Comment (shannona) [inline pages/browse-our-faq/bitmark-protocol-faq.md:41]: I'd suggest against using casual phrases like "BigTech" in docs. Perhaps "large, centralized tech firms" or something similar.

File: pages/browse-our-faq/bitmark-protocol-faq.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+An end-to-end protocol for securing the provenance of data is compatible and could help improve any local, government regulation by providing more transparency into the information. 
+
+## Why use a blockchain?
+
+Currently, the only technology we know that can secure the provenance of data without requiring a trusted third party is a public blockchain. 
+
+## How much does this cost?
+
+Right now it costs two times the bitcoin payment. We are working towards layer two options to radically lower this price.
+
```
Comment (shannona) [inline pages/browse-our-faq/bitmark-protocol-faq.md:55]: I don't understand what "two times the bitcoin payment" means. People will want more specific numbers.

File: pages/contribute-to-the-protocol.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+# Contribute to the Protocol
+_Last updated: 24-FEB 2020_
+
+[TOC]
+
+
+## About the Bitmark Protocol
+
+We expect you probably got here from https://bitmark.com.  But just in case you didn’t, the explanation of the Bitmark Protocol is here: 
+
```
Comment (shannona) [inline pages/contribute-to-the-protocol.md:20]: People might get here from searches. It's never good to presume how someone got to a web page. I'd suggest cutting this first paragraph (above the links), then sticking those links a little further into this section.

File: pages/contribute-to-the-protocol.md [hunk 7]
```diff
@@ -0,0 +1,151 @@
+---
+title: Contribute to the Protocol
+keywords: governance, security program, Bitmark Upgrade Proposal
+last_updated: 
+sidebar: mydoc_sidebar
+permalink: /contribute-to-the-protocol
+folder: 
+---
```
Comment (shannona) [inline pages/contribute-to-the-protocol.md:8]: It feels like this section has a lot about Bitmark governance, which isn't really on topic for "Contribute to the Protocol".

I'd suggest either breaking this section up (into governance [or about-the-company] and contributing). Or maybe renaming it, if there's something that's more obviously about both topics.

File: pages/explore-the-protocol-docs/blockchain-overview.md [hunk 8]
```diff
@@ -0,0 +1,314 @@
...
+keywords: blockchain
+last_updated: 
+sidebar: mydoc_sidebar
+permalink: /explore-the-protocol-docs/blockchain-overview
+folder: explore-the-protocol-docs
+---
+
+# Blockchain Overview
+
+This brief description of the structure of the structure of the Bitmark blockchain moves
```
Comment (shannona) [inline pages/explore-the-protocol-docs/blockchain-overview.md:12]: There's a duplicate here: "of the structure of the structure"

File: pages/explore-the-protocol-docs/glossary.md [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+**Bitmark API** — A programming interface that allows programmers to interface with the _Bitmark Property System_. It is an Open Rest API.
+
+**Bitmark blockchain** — The _blockchain_ used by Bitmark to record _Bitmark certificates_. It is both decentralized and distributed and is also open, public, and free. Also called the _Bitmark public blockchain_ or the _Bitmark public blockchain ledger_.
+
+**Bitmark certificate** — A unique identifier that provides a record of _property rights_ for _digital assets and data_. Technically, this is either an _issue record_ or a _transfer record_ that records the current ownership of a _digital property_, associated with an _asset record_ that describes what the digital property is.
+
+**Bitmark Health** — An iPhone app.
+
+**Bitmark Property System** — The _digital assets and data_ management system that consists of the _Bitmark blockchain_, the _Bitmark Account_, and other tools.
+
```
Comment (shannona) [inline pages/explore-the-protocol-docs/glossary.md:41]: I noted "Bitmark Protocol" as a new phrase in these docs, which should get a definition here.

**Review by shannona — APPROVED** [review]
It's a little hard to see how a big reorg like this works in diffs, but this generally looks like a good revision to me. I'm particularly happy with the collection of different articles into the explore-the-protocol section. I'd tried to move them a little closer together in my last line edits, but they really needed their own section, which you've provided.

I've offered a few comments here and there. The one place where I felt like I could only do part of the job was in the new FAQ section. I gave a few comments, but this is a new section, so I think I could add more value with a complete line edit. If you let me know when it goes into the master repo, I'll provide you a line edit as a PR.

**Comment by vienlt** [comment]
@moskovich: @shannona has some comments on the section name (register and control data), and the content (contribute to the protocol & FAQ). Could you take a look?

**Pull request merged**